clients/da-vinci-client/src/test/java/com/linkedin/davinci/store/rocksdb/RocksDBStoragePartitionTest.java

```diff
@@ -902,4 +904,83 @@ public void checkMemoryLimitAtDatabaseOpen() {
       removeDir(storeDir);
     }
   }
+
+  @Test
+  public void testCreateSnapshotWithBlobTransferEnable() {
+    String storeName = Version.composeKafkaTopic(Utils.getUniqueString("test_store"), 1);
```

Comment:
nit: maybe we can reuse some codes for these two methods so there's less duplicated codes